python/doc/theory/probabilistic_modeling/parametric_spectral_model.rst

```diff
@@ -22,15 +22,16 @@ Parametric spectral density functions
 
 **The Cauchy spectral model**
 
-Its is associated to the Exponential covariance model. The Cauchy spectral model is defined by:
+Its is associated to the Kronecker covariance model built upon an exponential covariance model (AbsoluteExponential). The Cauchy spectral model is defined by:
 
 .. math::
   :label: cauchyModel
 
-    S_{ij}(f) = \displaystyle \frac{4R_{ij}a_ia_j(\lambda_i+ \lambda_j)}{(\lambda_i+ \lambda_j)^2 + (4\pi f)^2}, \quad \forall (i,j) \leq d
+    S_{ij}(f) = \displaystyle 2 \Sigma_{ij}\prod_{k=1}^{n} \frac{\theta_k}{1 + (2\pi \theta_k f)^2}, \quad \forall (i,j) \leq d
 
-where :math:`\mat{R}`, :math:`\vect{a}` and :math:`\vect{\lambda}`
-are the parameters of the Exponential covariance model defined in
+where :math:`\mat{\Sigma}` is the covariance matrix of the Kronecker
+covariance model, :math:`\vect{\theta}` the scale parameters of
+the AbsoluteExponential covariance model defined in
 section [ParamStationaryCovarianceFunction]. The relation
 :eq:`cauchyModel` can be explained with the spatial covariance function
```

Comment:
What does "can be explained" mean? "can be derived from" perhaps? In any case, it is not very informative.

Reply:
I think the original test refers to the way we define this spectral model. It was the Fourier counterpart of the oldest `ExponentialModel`. `derived` is indeed more informative